src/System.Security.Cryptography.Cng/src/Microsoft/Win32/SafeHandles/NCryptSafeHandles.cs

```diff
@@ -53,42 +56,46 @@ private enum OwnershipState {
             Holder
         }
 
-        private OwnershipState m_ownershipState;
+        private OwnershipState _ownershipState;
 
         /// <summary>
         ///     If the handle is a Duplicate, this points at the safe handle which actually owns the native handle.
         /// </summary>
-        private SafeNCryptHandle m_holder;
+        private SafeNCryptHandle _holder;
 
-        protected SafeNCryptHandle() : base(IntPtr.Zero, true) {
+        protected SafeNCryptHandle() : base(IntPtr.Zero, true)
+        {
             return;
         }
 
         /// <summary>
-        ///     Wrapper for the m_holder field which ensures that we're in a consistent state
+        ///     Wrapper for the _holder field which ensures that we're in a consistent state
         /// </summary>
-        private SafeNCryptHandle Holder {
-            get {
-                Contract.Requires((m_ownershipState == OwnershipState.Duplicate && m_holder != null) ||
-                                  (m_ownershipState != OwnershipState.Duplicate && m_holder == null));
-                Contract.Requires(m_holder == null || m_holder.m_ownershipState == OwnershipState.Holder);
+        private SafeNCryptHandle Holder
+        {
+            get
+            {
+                Contract.Requires((_ownershipState == OwnershipState.Duplicate && _holder != null) ||
+                                  (_ownershipState != OwnershipState.Duplicate && _holder == null));
+                Contract.Requires(_holder == null || _holder._ownershipState == OwnershipState.Holder);
 
-                return m_holder;
+                return _holder;
             }
 
-            set {
-                Contract.Ensures(m_holder.m_ownershipState == OwnershipState.Holder);
-                Contract.Ensures(m_ownershipState == OwnershipState.Duplicate);
+            set
+            {
+                Contract.Ensures(_holder._ownershipState == OwnershipState.Holder);
+                Contract.Ensures(_ownershipState == OwnershipState.Duplicate);
 #if DEBUG
                 Contract.Ensures(IsValidOpenState);
                 Contract.Assert(value.IsValidOpenState);
 #endif
-                Contract.Assert(m_ownershipState != OwnershipState.Duplicate);
-                Contract.Assert(value.m_ownershipState == OwnershipState.Holder);
-                
-               
-                m_holder = value;
-                m_ownershipState = OwnershipState.Duplicate;
+                Contract.Assert(_ownershipState != OwnershipState.Duplicate);
```

Comment:
We've standardized across corefx on using Debug.Assert instead of Contract.Assert.